README.md

```diff
@@ -66,7 +66,7 @@
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/217890917-605fb6a5-c8fe-4dee-904a-ef546f423f1a.png)

- [ ] You are missing ESLint from your linters setup which checks your JavaScript files. It checks those errors which can break your application :thinking: